docs/develop/serpentVersions.rst

```diff
@@ -125,3 +125,31 @@ layout of output files, such that present readers will not work, then
 version-specific read methods will have to be written.
 If/when this happens, a procedure will be developed and added here.
 
+MapStrVersions
+----------------
+
+This is a mapping variable located in the ResultsReader parser. 
+The variable is a nested dictionary, where the keys describe the version and the nested
+dictionary describes the various data blocks within the results file (_res.m). 
+
+In general the results file is divided into three main blocks:
+- metadata: general description of the simulation
+- resdata: time-dependent values, e.g. k-eff
+- universes: universe dependent values, such as cross-sections
+The mapping through the variable `MapStrVersions` should reflect this. 
```

Comment:
For rst, you have to use double ticks to get the code formatted properly - http://docutils.sourceforge.net/docs/ref/rst/roles.html#literal